src/webextension/widgets/filter-input.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import PropTypes from "prop-types";
+import React from "react";
+
+import styles from "./filter-input.css";
+
+export default class FilterInput extends React.Component {
```

Comment:
While its fresh, is now an good time to consider adding jsdoc comments? Or do you want to go back through everything separately and catch it all at once?

Reply:
Per our Vidyo chat, let's do JSDocs after alpha.